Sources/Danger/GitLabDSL.swift

```diff
@@ -106,14 +106,14 @@ extension GitLab {
             }
 
             public enum CodingKeys: String, CodingKey {
-                case id // swiftlint:disable:this identifier_name
+                case id 
                 case ref
                 case sha
                 case status
                 case webUrl = "web_url"
             }
 
-            public let id: Int // swiftlint:disable:this identifier_name
+            public let id: Int 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitLabDSL.swift;
  Line: 116;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)